server.js

```diff
@@ -1,30 +1,192 @@
-import express from "express";
-import cors from "cors";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+import express from 'express';
+import cors from 'cors';
+import netflixData from './data/netflix-titles.json';
+import MarkdownIt from 'markdown-it';
+
+// Defines the port the app will run on. Defaults to 8080.
 const port = process.env.PORT || 8080;
-const app = express();
 
-// Add middlewares to enable cors and json body parsing
+// Import a module for listing all endpoints in the app
+const app = require('express')();
+const md = new MarkdownIt();
+
+// Add middlewares to enable cors and json body parsing 
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+
+// Default route to list all available endpoints in the app
+app.get('/', (req, res) => {
+    // MarkdownIt is a library for converting Markdown to HTML.
```

Comment:
This was fun! That you used a library to show all the endpoints in a nice way and well done for going for something else than suggested in the instructions! You transformed the boring "backend-look" into a front-end look that is a lot easier to read!